jobs/config.json

```diff
@@ -3233,6 +3233,30 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-coredns": {
+    "args": [
+      "--cluster=gce-scale-cluster",
+      "--env=CLUSTER_DNS_CORE_DNS=true",
+      "--env=HEAPSTER_MACHINE_TYPE=n1-standard-8",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-large-performance.env",
```

Comment:
We don't want this.

Reply:
+1. And I think all those 3 env files are irrelevant for this scale.